src/assets/css/sections/hero_section.css

```diff
@@ -0,0 +1,34 @@
+.hero_section {
+  display: flex;
+  flex-direction: row;
+  margin-top: 25px;
+  margin-left: 145px;
+  margin-right: 150px;
+}
+.hero_section_left {
+  padding: 18px;
+  margin-top: 107px;
```

Comment:
Can you help explain why do we need use margin here?

Reply:
Because it separates hero_section_left from hero_section, if I remove it the hero_section_text will be on top and not like the design